dr_evaluation/app.py

```diff
@@ -0,0 +1,251 @@
+
+    return model
+
+# TODO: Make a featurize function to work for fitting and predicting
+ 
+def predict(model, site, event_day, cli):
+    '''
+    Takes a model from the "fit" function and predicts for the event day
+    cli: pymortar client
+    '''
+    start, end = get_window_of_day(event_day)
+    interval = '15m'
+    agg = 'MEAN'
+    alphas = [.001, 0.01, 0.05, 0.1, 0.5, 1, 10]
+
+    # Get weather
+    weather = gd.get_weather(site, start, end, agg=agg, window=interval, cli=cli)
+    weather.index = pd.date_range(start, end, freq='15min')[:-1]
+    weather = weather.interpolate()
+    closest_station = get_closest_station(site)
+    if closest_station is not None:
+        weather = pd.DataFrame(weather[closest_station])
+    else:
+        weather = pd.DataFrame(weather.mean(axis=1))
+
+
+    # Get power
+    power = gd.get_power(site, start, end, agg=agg, window=interval, cli=cli) * 4
+
+    # Merge
+    weather_mean = pd.DataFrame(weather.mean(axis=1))
+    power_sum = power.sum(axis=1)
+    try:
+        power_sum.index = pd.date_range(start, end, freq='15min')[:-1]
+    except:
+        # handling error with one less value in power
+        print('error in power index')
+        power_sum = power_sum.append(pd.Series([power.values[-1]]))
+        power_sum.index = pd.date_range(start, end, freq='15min')[:-1]
+    power_sum = pd.DataFrame(power_sum)
+    data = power_sum.merge(weather_mean, left_index=True, right_index=True)
+    data.columns = ['power', 'weather']
+
+    data['date'] = data.index.date
+    df = data
+
+    # Get time of week
+    df['time_of_week'] = [get_time_of_week(t) for t in df.index]
+    decoy = pd.DataFrame({
+    'time_of_week':np.arange(0, 480)
+    })
+    df = df.append(decoy, sort=True)
+    indicators = pd.get_dummies(df['time_of_week'])
+    df = df.merge(indicators, left_index=True, right_index=True)
+    df = df.drop(labels=['time_of_week'], axis=1)
+    df = df.iloc[:-480]
+
+    # Get changes in weather from last 15 minutes
+    df['change'] = (df['weather'] - np.roll(df['weather'], 1))
+    
+    # Get temperature cutoffs
+    cutoffs = [40, 50, 60, 70, 80]
+    arr = df['weather'].apply(lambda t: get_t_cutoff_values(t, cutoffs)).values
+    a = np.array(arr.tolist())
+    t_features = pd.DataFrame(a)
+    t_features.columns = ['temp_cutoff_' + str(i) for i in cutoffs] + ['max_cutoff']
+    t_features.index = df.index
+    df = df.merge(t_features, left_index=True, right_index=True)
+
+    # Take away power from predicting data
+    X_test, actual = df.drop(['power', 'date', 'weather'], axis=1), df['power']
+
+    # Predict and find test error
+    baseline = model.predict(X_test)
+    baseline = pd.Series(baseline, index=actual.index)
+
+    return actual, baseline
+
+if __name__ == '__main__':
+    from pge_events import pge_events
+    
+    with open('config.json') as f:
+        config = json.load(f)
+
+    # start pymortar client
+    client = pymortar.Client({
```

Comment:
I think it would be more consistent to just use the `pymortar.Client({})` constructor and use the environment defaults, rather than asking users to write down their username and password in the file